solr/bin/solr

```diff
@@ -2073,8 +2073,8 @@ else
 fi
 
 # IP-based access control
-IP_ACL_OPTS=("-Dsolr.jetty.inetaccess.includes=${SOLR_IP_WHITELIST}" \
-             "-Dsolr.jetty.inetaccess.excludes=${SOLR_IP_BLACKLIST}")
+IP_ACL_OPTS=("-Dsolr.jetty.inetaccess.includes=${SOLR_IP_ALLOWLIST}" \
+             "-Dsolr.jetty.inetaccess.excludes=${SOLR_IP_DENYLIST}")
```

Comment:
I could have tried to support both in parallel for some time and warned when detecting the old, but it's a hassle across bash and Windows - any strong feelings about this? We spell it out in upgrade notes...

Reply:
On second thought, since this is about security (IP filtering), I plan to make the bin/solr scripts print an error and exit if the old env.var is set. That is fail-fast, and users can re-configure to use the new variable names and start again. Already tested a fix for linux. Need to test same for bin/solr.cmd

Reply:
Implemented and tested also for Windows.